src/renault_api/kamereon/models.py

```diff
@@ -49,13 +49,17 @@
     "X101VE": {  # ZOE phase 1
         "reports-in-watts": True,
         "support-endpoint-location": False,
+        "support-endpoint-lock-status": False,
     },
     "X102VE": {  # ZOE phase 2
         "warns-on-method-set_ac_stop": "Action `cancel` on endpoint `hvac-start` may not be supported on this model.",  # noqa
     },
     "XJB1SU": {  # CAPTUR II
         "support-endpoint-hvac-status": False,
     },
+    "XJD1SU": {  # NEW DUSTER
+        "support-endpoint-hvac-start": False,
+    },
```

Comment:
I think this shouldn't be mixed in with "lock_status" updates.
It should be kept for another PR.